resources/views/checkout/steps/login.blade.php

```diff
@@ -1,4 +1,4 @@
-<checkout-login v-slot="checkoutLogin">
+<checkout-login v-slot="checkoutLogin" next-url="{{ route('checkout', ['step' => 'credentials']) }}">
```

Comment:
This responsibility should fall within the [login page](https://github.com/rapidez/core/blob/master/resources/views/checkout/pages/login.blade.php)

Doing this in the step causes it to happen on onepage checkouts or other combinations (like done in the checkout theme) as well